syntax = "proto3";

message NoiseMeasurement {
  string device_name = 1;
  float noise_value = 2;
}
